src/numeric/grids.jl

```diff
@@ -5,19 +5,48 @@ function mlinspace(min, max, n)
     return QE.gridmake(nodes...)
 end
 
-type CartesianGrid <: Grid
-    min::Array{Float64,1}
-    max::Array{Float64,1}
-    n::Array{Int64,1}
+type EmptyGrid <: Grid
```

Comment:
Sure.
But... here we are back to the question of what is the good way to add cached data to an object. In this case, it might be that constructing the grid is costly and you don't want to do it by default. But when you construct it you want to keep it. Then, hypothetically, what is the right way to proceed ? Subtype ? Create another nutable type with a grid field, a fillable nodes slot ?